src/resources/parser/texture/hdr.js

```diff
@@ -1,54 +1,13 @@
-import { http } from '../../../net/http.js';
+import { Asset } from '../../../asset/asset.js';
+import { Texture } from '../../../graphics/texture.js';
+import { ReadStream } from '../../../core/read-stream.js';
```

Comment:
Hmm, I'm thinking we might need to write up some rules about ordering imports. When I originally ported the engine codebase to ES6 modules, the ordering was based on the root folders in src and something like:

core
math
net
graphics
scene
resources
asset
framework

(I've left a few out but you get the idea)

Within each folder group, I just ordered alphabetically. To me, this felt easier to spot circular dependency problems, where a low layer (e.g. graphics) might import from a higher layer (e.g. scene).

And I'd typically put a blank line between each group.

Look at `application.js` for a big example.

In that world, you'd have:

```javascript
import { ReadStream } from '../../../core/read-stream.js';

import { Texture } from '../../../graphics/texture.js';

import { Asset } from '../../../asset/asset.js';
```

Reply:
I've updated the order and spacing.